test/hotspot/jtreg/compiler/intrinsics/base64/TestBase64.java

```diff
@@ -177,4 +236,139 @@ private static void assertEqual(Object result, Object expect) {
             }
         }
     }
+
+    private static byte[] hexStringToByteArray(String s) {
```

Comment:
A method to convert a Hex String to a byte array already exists in test/lib/jdk/test/lib/Utils.java  in  toByteArray(s).
Add "jdk.test.lib.Utils" to the "@build" line at the top of the test.

Reply:
Cool! Thanks.  I was unaware of that function.  Will fix.